data/json/effects.json

```diff
@@ -506,6 +507,7 @@
     "remove_message": "The weight on your chest lessens.",
```

Comment:
Properly shocky patients *wish* they were dizzy. Shortness of breath, chest pains and throwing up is their usual fare.
I'm not fond of the description, but I wanted to keep it vague enough that I can still use it as a blanket term for different venoms (bees and wasps use histamine and catecholamines, spiders deal mostly in protein toxins).

Reply:
Hmm... I see. (Catecholamines as an overall _vasodilator_, though? I suppose if the histamine effects predominated it could happen. An epinephrine injection is normally rather useful in _**non-bleeding**_ shock...)

Very little is known about jumping spider venoms; one recent survey found a total of 4 species with any protein sequences known. Based on the symptoms of jumping spider bites, treating their venom as histamine-like makes sense - any neurotoxic components may well be specialized for insects. (Possibly "Systemic Inflammatory Agents"?)

Reply:
Oh - one note. Asthma is going to cause someone to be very susceptible to such toxins - OTOH, their inhalers will actually likely help anyone who uses them...

Reply:
Histamine is the dominant bioactive amine in most cases, but there's enough (nor)adrenaline and dopamine mixed in there to give you that warm fuzzy feeling of all vasomotoric receptors getting pushed at once.
I could see Inflammatory Agents working, tbh. Inflammation is widely recognizable as a bad thing to have, and it's open enough to fudge the details if that's necessary. Thanks!
Overhauling trait-effect interactions is way beyond my cargoculting abilities, so that would require someone actually competent down the line.

Reply:
Quite welcome; happy to help! (I'm thinking the "shocky patients" you mentioned are anaphylaxis-type shock? I was going purely off of vasodilation and thinking hypotension...)

I plan to be doing at least a bit of C++ tinkering (as well as JSON) when I can finally move enough stuff off of my main hard drive - git repositories, in my experience, consume rather more space than the current working version. (Admittedly, the repository I'm thinking of is the one for Python...) Hope I can manage it before I need to go back to teaching!

Reply:
Most shock forms end in the same clinical presentation, so those symptoms are pretty general. Strictly speaking an anaphylactic reaction would require a preexisting allergy, but since there's a lot of histamine involved in these it's not a bad jump-off point. Thankfully we don't yet have 80kg wasps roaming the countryside so it's all conjecture, but the realistic effect would not be far off. 
I decided to not have adrenaline shots work as a direct antivenom mostly to keep the exact mechanism vague enough to be able to recycle the venom effects for unrelated critters.